cpp/src/gandiva/tree_expr_builder.h

```diff
@@ -85,6 +86,25 @@ class TreeExprBuilder {
   /// \brief convenience function for simple function conditions.
   static ConditionPtr MakeCondition(const std::string& function,
                                     const FieldVector& in_fields);
+
+  /// \brief creates an in expression
+  static NodePtr MakeInExpressionInt(NodePtr node,
+                                     const std::unordered_set<int32_t>& constants);
+  static NodePtr MakeInExpressionLong(NodePtr node,
```

Comment:
I'm not sure why those are suffixed `Int`/`Long` rather than `Int32`/`Int64`? A Java reference perhaps? :-)

Reply:
ha yes..old habits :) will change them